### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by tom-essen**

### CODE REVIEW

File: staff/tomas-sancheznavarro/pre-course/calculator-console-pro/calculadora PRO.js [hunk 1]
```diff
@@ -0,0 +1,68 @@
+function calculate() {
```
Comment (diesttro) [inline staff/tomas-sancheznavarro/pre-course/calculator-console-pro/calculadora PRO.js:1]: file name with space!

File: staff/tomas-sancheznavarro/pre-course/calculadora-html-js/main.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
...
+
+    if (operator == "add") {
+        calculate = a + b;
+    } else if (operator == "min") {
+        calculate = a - b;
+    } else if (operator == "div") {
+        calculate = a / b;
+    } else if (operator == "mul") {
+        calculate = a * b;
+    }
```
Comment (aleixcam) [inline staff/tomas-sancheznavarro/pre-course/calculadora-html-js/main.js:15]: switch case is a better option here

File: staff/tomas-sancheznavarro/pre-course/project-skylab-airlines/skyLabAirlines.js [hunk 3]
```diff
@@ -0,0 +1,179 @@
...
+        greeting = "Buenas noches";
+    }
+
+    return greeting;
+}
+
+var resultTime = dayHour();
+
+function logIn() {
+    const name = prompt("Ingrese su nombre");
```
Comment (Sergioluz90) [inline staff/tomas-sancheznavarro/pre-course/project-skylab-airlines/skyLabAirlines.js:109]: you may comprobe if it is empty

File: staff/tomas-sancheznavarro/cv/index.html [hunk 4]
```diff
@@ -0,0 +1,81 @@
...
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
+</head>
+
+<body>
+    <header>
+        <h1>Juan de los Palotes | Web Developer</h1>
+        <hr>
+    </header>
+    <div>
```
Comment (HenryNovoa) [inline staff/tomas-sancheznavarro/cv/index.html]: div not semantic!!

File: staff/tomas-sancheznavarro/cv/index.html [hunk 5]
```diff
@@ -0,0 +1,81 @@
...
+        <address>
+            Carrer de la Unió 163<br>Premià de Mar<br>Barcelona<br>
+            <a href="mailto:palotes.juan@jeemail.com">palotes.juan@jeemail.com</a>.<br>
+            <hr></address>
+    </div>
+
+    <section id="goals">
+        <h2><u>Goals</u></h2>
+        <blockquote>
```
Comment (seand52) [inline staff/tomas-sancheznavarro/cv/index.html]: blockquote is used to quote things from other sources, like a book quote. In this case I think you can just use a <p> tag

Comment (Pacoscript) [inline staff/tomas-sancheznavarro/cv/index.html]: por qué utilizaste blockquote??? no entiendo

File: staff/tomas-sancheznavarro/cv/index.html [hunk 6]
```diff
@@ -0,0 +1,82 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Document</title>
```
Comment (gbaste) [inline staff/tomas-sancheznavarro/cv/index.html]: Define a title for your HTML document

https://www.w3schools.com/tags/tag_title.asp

File: staff/tomas-sancheznavarro/cv/index.html [hunk 7]
```diff
@@ -0,0 +1,82 @@
...
+        </ul>
+        <h3>Frameworks</h3>
+        <ul>
+            <li>React.js</li>
+            <li>Vue.js</li>
+            <li>Node.js</li>
+        </ul>
+    </section>
+
+    <section id="education">
```
Comment (gbaste) [inline staff/tomas-sancheznavarro/cv/index.html]: the id is not necessary, you must remove it

File: staff/tomas-sancheznavarro/pre-course/calculadora-html-js/main.js (hunk 2 above)
Comment (tom-essen) [reply staff/tomas-sancheznavarro/pre-course/calculadora-html-js/main.js:15]: done!

File: staff/tomas-sancheznavarro/pre-course/project-skylab-airlines/skyLabAirlines.js (hunk 3 above)
Comment (tom-essen) [reply staff/tomas-sancheznavarro/pre-course/project-skylab-airlines/skyLabAirlines.js:109]: done!

File: staff/tomas-sancheznavarro/cv/index.html (hunk 6 above)
Comment (tom-essen) [reply staff/tomas-sancheznavarro/cv/index.html]: done!

File: staff/tomas-sancheznavarro/cv/index.html (hunk 7 above)
Comment (tom-essen) [reply staff/tomas-sancheznavarro/cv/index.html]: done!

**Pull request merged**